config/scout.php

```diff
@@ -141,7 +141,7 @@
         'index-settings' => [
             Game::class => [
                 'filterableAttributes' => ['id', 'title'],
-                'searchableAttributes' => ['title', 'id'],
+                'searchableAttributes' => ['title', 'alt_titles', 'id'],
```

Comment:
More of a future problem, but when the match is only on the alt_title, it's hard to tell why the record matched.

For example, if I add an alternate title for Sonic the Hedgehog as "Speedy Blue Devil" (as maybe some made-up release for Africa) and search on "Speedy", I get a bunch of exact matches against Speedy Gonzales, then the Sonic game, and then some light matches against Speed, which is logically correct. But if the alternate title is not well known, wouldn't seeing "Sonic" in the middle of that list confuse the user?

Reply:
Yes, we'll have to figure out something for this when we expose the search results to the UI, which shouldn't be too long from now.